### DISCUSSION TITLE
dbeaver community 24.3版本，分屏及汉化的问题

### DISCUSSION THREAD

**Discussion by xiayutian** [dbeaver/dbeaver#36599] [Q&A]
1、dbeaver community 24.3版本，sql编辑器垂直分屏后，左右编辑区域互相影响，sql控制台模式，分配是正确的。
2、软件已经选择了简体中文，但是菜单里面还有很多是英文，汉化不彻底。如图
![image](https://github.com/user-attachments/assets/c702debd-f78b-48e1-bb11-3c077c65e3eb)

**Comment by EastLord** [comment]
等我有时间
